pkg/provider/azure_loadbalancer.go

```diff
@@ -823,6 +835,12 @@ func (az *Cloud) ensurePublicIPExists(service *v1.Service, pipName string, domai
 		}
 		pip.Name = to.StringPtr(pipName)
 		pip.Location = to.StringPtr(az.Location)
+		if az.HasExtendedLocation() {
+			pip.ExtendedLocation = &network.ExtendedLocation{
+				Name: &az.ExtendedLocationName,
+				Type: &az.ExtendedLocationType,
+			}
+		}
```

Comment:
If I bind a service to an existing pip, the pip would not have the extended location information on itself. Is this an expected behaviour?

Reply:
Same qustion here, would it fail or still succeed in such a scenario?

Reply:
Just a little context, when creating an AKS cluster, the user will choose the extended location, and the whole the cluster will be created in the edge zone. So all the initial resources will have extended location fields set. I am assuming that by existing PIP, you mean the PIP created by AKS? 

In any case, there are 2 creators, AKS RP or cloudprovider, in both cases extended Location will be set.

Reply:
yeah I mean the unmanaged cluster created by aks-engine instead of aks

Reply:
We decided that AKS-engine will not support edge zones. That means the cloud config will never have the extended location properties set so we will never be in the situation

Reply:
Looks good. Could you squash the commits?

Reply:
Will do